### DISCUSSION TITLE
Creating copy of existing  Milvus database

### DISCUSSION THREAD

**Discussion by parthplc** [milvus-io/milvus#7421] [Q&A and General discussion]
Is there a way we can create a deep copy of the existing collection DB and make changes to the new one without setting it up from start? Like the same collection has two copies with different name

**Comment by yhmo** [comment]
No. v1.x doesn't provide such api to do copy, neither the v2.0. However I think this is a useful function, but its priority is not high. There is already an issue for this: https://github.com/milvus-io/milvus/issues/3237
Note: Currently we only fix critical issue for v1.x, basically we don't add new feature to 1.x. All new features are designed on v2.0.
